docs/sdd-workflow.md

```
@@ -12,11 +12,28 @@ them by hand.
 
 ```text
 research  →  scoping decision  →  spec  →  implementation plan  →  code
-(optional)   (optional)           (required for behaviour changes)
+(optional)   (optional*)          (required for behaviour changes)
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)